### DISCUSSION TITLE
Missing API Key feature that was introduced in v0.1.117?

### DISCUSSION THREAD

**Discussion by Kanishk-Kumar** [open-webui/open-webui#3935] [General]
# Bug Report

I'm unable to find/generate the keys in newer versions. This feature was introduced [here](https://github.com/open-webui/open-webui/releases/tag/v0.1.117). Where did it go?

## Logs and Screenshots

**Screenshots (if applicable):**
![image](https://github.com/user-attachments/assets/48b865db-1bac-4ff7-bc14-3df51c597fdb)

**Comment by justinh-rahb** [comment]
Huh? You posted a screenshot... with a big red arrow pointing at the very thing you claim you can't find...

<img width="666" alt="Screenshot 2024-07-17 at 5 38 51 PM" src="https://github.com/user-attachments/assets/a1208fa9-a6ee-41fd-a0f1-251fd96ad41e">

Did you try clicking any of the above buttons I've pointed out?

**Reply by Kanishk-Kumar** [reply]
Screenshot is from v0.1.117. Those buttons/section (for API Key) are not available in the last few versions. Please check for the same in the latest version. Thanks.

**Reply by justinh-rahb** [reply]
It is in latest, from where do you think I got **my** screenshot?

https://github.com/user-attachments/assets/eabc38fb-d1d6-490f-ba6a-3415c99537d6
